modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.tsx

```diff
@@ -35,18 +34,16 @@ export default function EditObjectFolder({
 		dispatch,
 	] = useFolderContext();
 
-	const [showModal, setShowModal] = useState<ViewObjectDefinitionsModals>({
+	const [showModal, setShowModal] = useState<ModelBuilderModals>({
 		addFolder: false,
 		addObjectDefinition: false,
-		bindToRootObjectDefinition: false,
```

Comment:
@igor-franca @rebsilva is it safe to remove this property and the "unbind" one?

Reply:
These three properties belong to the Root Concept, in [creating](https://github.com/liferay-objects/liferay-portal/pull/2524/commits/590ef9aacd411d05f16e9896361c10a51c001600#diff-63608e745a4c720211693896469a7bf97d034829944b98edab0d2ed29a092d7cR53-R58) the ModelBuilderModals interface I omitted these three properties.

Reply:
The reason for bringing this commit to this PR was to not add Root Concept properties within ModelBuilder

Reply:
Oh, I understand. Thanks for the explanation :)